launch/app/runtime/conf/smarthome.cfg

```diff
@@ -1,9 +1,9 @@
-org.eclipse.smarthome.folder:items=items
-org.eclipse.smarthome.folder:sitemaps=sitemap
-org.eclipse.smarthome.folder:rules=rules
-org.eclipse.smarthome.folder:scripts=script
-org.eclipse.smarthome.folder:persistence=persist
-org.eclipse.smarthome.folder:things=things
+org.openhab.core.folder:items=items
```

Comment:
This actually shows me that there was an error in the refactoring. The namespace for all configs has to be `org.openhab.<configname>`, so the `core` is too much in there.
@cweitkamp Can you fix this in the core?

Reply:
Yes, I will do that. And update #1030 accordingly.

Reply:
Done. See https://github.com/openhab/openhab-core/pull/1314.